codeworld-api/src/CodeWorld/Driver.hs

```diff
@@ -1835,8 +1840,6 @@ unsafeCollaborationOf numPlayers initial step event draw = do
     dhash <- getDeployHash
     let token = PartialToken dhash
     runGame token numPlayers initial step event draw
-  where
-    token = NoToken
```

Comment:
I hope I have removed the right one of the two definitions.

Reply:
I'm not sure anyone will notice the difference.  If the token is `PartialToken dhash`, then the server will prevent anyone from joining a multiplayer game session if they are running a different binary.  That's usually desirable, so let's go with this.  Is NoToken used anywhere, then?

Reply:
> Is NoToken used anywhere, then?

I've removed it now.